LocalLang/Typing.lean

```diff
@@ -1,7 +1,23 @@
 import LocalLang.AST
 import LocalLang.Types
 
+-- in context Γ expression ε has Type T
 -- TODO: define
-inductive TypeJdg : TypeContext → Expr → LLType → Prop where
 
+mutual
+  inductive TypeJdg : TypeContext → Expr → LLType → Prop where
+    | Jdg_Var {Γ : TypeContext}  {name : String}
+              (H : Γ.find? (·.1 = name) = some ⟨name, LLType.nat⟩)
```

Comment:
I would recommend using propositions over equalities.  In this case, you could do:
`(name, type) ∈ Γ`.